be/src/vec/columns/column.h

```diff
@@ -286,6 +299,12 @@ class IColumn : public COW<IColumn> {
     ///  passed bytes to hash must identify sequence of values unambiguously.
     virtual void update_hash_with_value(size_t n, SipHash& hash) const = 0;
 
+    /// Update state of hash function with value of n elements to avoid the virtual function call
+    /// null_data to mark whether need to do hash compute, null_data == nullptr
+    /// means all element need to do hash function, else only *null_data != 0 need to do hash func
+    virtual void update_hashes_with_value(std::vector<SipHash>& hash,
```

Comment:
why not make update_hashes_with_value just a virtual function, not a pure virtual function, and give a default implement in class Column?

Reply:
Here is the estimated design so that the subsequent implementation of the column can consider whether to implement this function, to save the omission of the error caused by